server/index.js

```diff
@@ -3,12 +3,15 @@ const app = express();
 const port = 3000;
 const morgan = require('morgan');
 const path = require('path');
+const db = require('../db.js');
 
 app.use(express.static(path.join(__dirname, '../public')));
 
-// app.get('/', (req, res) => {
-//   res.send();
-// });
+app.get('/reviews', (req, res) => {
+  db.getReviews((err, data) => {
+    res.send(data);
```

Comment:
if(err) {
console.log(error);
res.status(500).send();
} else {
res.status(200).send(data);
}
//Look out for error handling and put status codes for best practice.